Survey/Survey/Sources/Presentation/Modules/Login/LoginView.swift

```diff
@@ -0,0 +1,66 @@
+//  swiftlint:disable closure_body_length
+//
+//  LoginView.swift
+//  Survey
+//
+//  Created by Taher on 2/11/22.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+
+    @State private var logoOffset = 0.0
+    @State private var logoScaleEffect = 1.0
+    @State private var loginViewOpacity = 0.0
+    @State private var isLoginFailed = false
+    @State var editingMode: Bool = false
+    @State var email: String = ""
+    @State var password: String = ""
+
+    var body: some View {
+        GeometryReader { _ in
+            ZStack {
+                ImageAssets.loginBackground.imageView
+                    .resizable()
+                    .aspectRatio(contentMode: .fill)
+                    .edgesIgnoringSafeArea(.all)
+                OverlayView.loginScreen.linearGradient
+                ImageAssets.logoWhite.imageView
+                    .resizable()
+                    .frame(width: 202.0, height: 48.0, alignment: .center)
+                    .scaleEffect(logoScaleEffect)
+                    .offset(y: editingMode ? -340 : logoOffset)
+                VStack(alignment: .center, spacing: 20.0) {
+                    EmailTextField(email: $email, editingMode: $editingMode)
+                    PasswordSecureTextField(password: $password, editingMode: $editingMode)
+                    Button(
+                        action: {
+                            // To-Do: send action to viewmodel
```

Comment:
```suggestion
                            // TODO: send action to viewmodel
```

Reply:
update in this [commit](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/41/commits/f10d87bdfe1267e19fe24bb94768bd66860444a7)